packages/lib/checkRateLimitAndThrowError.ts

```diff
@@ -55,6 +55,9 @@ async function changeSMSLockState(identifier: string, status: SMSLockState) {
   }
 
   if (userId) {
+    const user = await prisma.user.findUnique({ where: { id: userId, profiles: { none: {} } } });
+    if (status === SMSLockState.REVIEW_NEEDED && user?.smsLockReviewedByAdmin) return;
```

Comment:
```suggestion
    if (status === SMSLockState.UNLOCKED && user?.smsLockReviewedByAdmin) return;
```

If it was unlocked by  admin then we don't need to mark it for review or locked again

Reply:
I think this one needs to be `if (user?.smsLockReviewedByAdmin) return;` by removing `SMSLockState.UNLOCKED` condition because as you can see on changeSMSRateLimit function. The rate limiting type is either LOCKED or REVIEW_NEED so no need to check for UNLOCKED.